### PR TITLE
LW - R7: Editor doesn't refresh after complete task (LEAN-1848)

### PR DESCRIPTION
**Pull request by asouqi**

### CODE REVIEW

File: src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx [hunk 1]
```diff
@@ -27,25 +27,19 @@ import {
...
 import React, { useEffect, useMemo } from 'react'
 import styled from 'styled-components'
 
 import config from '../../../config'
 import { useCreateEditor } from '../../../hooks/use-create-editor'
 import useTrackedModelManagement from '../../../hooks/use-tracked-model-management'
-import {
-  graphQLErrorMessage,
-  Person,
-  useGetPermittedActions,
```
Comment (mbartenev-atypon) [inline src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx:41]: I don't think there is a good reason not to delete these graphql things. We are planning to stop direct contact from graphql server anyway.

File: src/index.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ import { ThemeProvider } from './theme/ThemeProvider'
 export { ProjectRole } from './lib/roles'
 export type { state } from './store'
 export * from './store/ParentObserver'
+export { getUserRole } from './lib/roles'
```
Comment (mbartenev-atypon) [inline src/index.tsx:35]: Maybe I'm missing something but I don't see it being used.

**Review by mbartenev-atypon — CHANGES_REQUESTED** [review]

File: src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx (hunk 1 above)
Comment (asouqi) [reply src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx:41]: I did a clean-up there, for unused queries

File: src/index.tsx (hunk 2 above)
Comment (asouqi) [reply src/index.tsx]: removed

**Review by mbartenev-atypon — APPROVED** [review]

**Pull request merged**